calculators/memoria_rating.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+$GLOBALS['rating_calculators']['memoria_rating'] = [
+	'description' => '
+		<a href="/player/125980">Memoria\'s</a> rating, v1.0
+
+		Based on the main ratings.
+		- Starting rating is 2500 instead of 2000, minimum is capped at 100, and rating change ranges from 0 to 50, or 100 in case of being afk.
+		- Factor is changed from 4000 to 2000, meaning that at 4500 you have a 90% prediction to win against a 2500.
+		- Games under 3m40s are not counted towards rating.
+		- Only games from Standard and (Heroic) Commanders gamemodes are included.
+		- Team rating is based on the highest rated player and the lowest rated player.
+		  - Generally team rating is top player\'s rating.
+		  - If the lowest rated player on a team has rating lower than 33% of the highest rated player, team rating is lowered slightly.
```

Comment:
how exactly "slightly"? as a user, I'd be interested in this (and as a coder, I can't grasp wtf is `ceil(sqrt($team_rating[0]**2 - $team_rating_low[0]**2)**0.75)` at a glance).
Give your great examples like you did with "factor" if you can.

also notice how 33*3 != 100, you may want to say it the way you have it in code, "if highest rated player rating is 3+ times higher than lowest"